spec/factories/book.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :book do
+    code { Faker::Internet.uuid }
+    name { Faker::Name.first_name }
+    description { Faker::Lorem.sentence }
+    price { Faker::Number.decimal(l_digits: 4, r_digits: 2) }
+    number_of_pages { Faker::Number.number(digits: 3) }
+    date_published { Faker::Date.in_date_period }
+    active { [true, false].sample }
```

Comment:
Remove this field from factory. In case you need an active book in your test, just set it there.